### ISSUE TITLE
Only video extracted from m3u8

### ISSUE THREAD

**Issue by Ethemind** [ytdl-org/youtube-dl#25234]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2020.05.08**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
PASTE VERBOSE LOG HERE
```
youtube-dl -v --all-subs https://live.performa.intio.tv/media/0af0186b-112c-4e72-8556-54abedcc7f55/master-allsubs.m3u8
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-v', u'--all-subs', u'https://live.performa.intio.tv/media/0af0186b-112c-4e72-8556-54abedcc7f55/master-allsubs.m3u8']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2020.05.08
[debug] Python version 2.7.17 (CPython) - Linux-4.15.0-99-generic-x86_64-with-LinuxMint-19-tara
[debug] exe versions: ffmpeg 3.4.6, ffprobe 3.4.6, phantomjs 5, rtmpdump 2.4
[debug] Proxy map: {}
[generic] master-allsubs: Requesting header
[generic] master-allsubs: Downloading m3u8 information
[debug] Default format spec: bestvideo+bestaudio/best
[debug] Invoking downloader on u'https://live.performa.intio.tv/media/0af0186b-112c-4e72-8556-54abedcc7f55/hls_video_6_7800_25.m3u8'
[download] Destination: master-allsubs-master-allsubs.f8685.mp4
[debug] ffmpeg command line: ffmpeg -y -loglevel verbose -headers 'Accept-Charset: ISO-8859-1,utf-8;q=0.7,*;q=0.7
Accept-Language: en-us,en;q=0.5
Accept-Encoding: gzip, deflate
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3530.3 Safari/537.36
' -i 'https://live.performa.intio.tv/media/0af0186b-112c-4e72-8556-54abedcc7f55/hls_video_6_7800_25.m3u8' -c copy -f mp4 'file:master-allsubs-master-allsubs.f8685.mp4.part'
ffmpeg version 3.4.6-0ubuntu0.18.04.1 Copyright (c) 2000-2019 the FFmpeg developers
  built with gcc 7 (Ubuntu 7.3.0-16ubuntu3)
  configuration: --prefix=/usr --extra-version=0ubuntu0.18.04.1 --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --enable-gpl --disable-stripping --enable-avresample --enable-avisynth --enable-gnutls --enable-ladspa --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librubberband --enable-librsvg --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzmq --enable-libzvbi --enable-omx --enable-openal --enable-opengl --enable-sdl2 --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-chromaprint --enable-frei0r --enable-libopencv --enable-libx264 --enable-shared
  WARNING: library configuration mismatch
  avcodec     configuration: --prefix=/usr --extra-version=0ubuntu0.18.04.1 --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --enable-gpl --disable-stripping --enable-avresample --enable-avisynth --enable-gnutls --enable-ladspa --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librubberband --enable-librsvg --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzmq --enable-libzvbi --enable-omx --enable-openal --enable-opengl --enable-sdl2 --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-chromaprint --enable-frei0r --enable-libopencv --enable-libx264 --enable-shared --enable-version3 --disable-doc --disable-programs --enable-libopencore_amrnb --enable-libopencore_amrwb --enable-libtesseract --enable-libvo_amrwbenc
  libavutil      55. 78.100 / 55. 78.100
  libavcodec     57.107.100 / 57.107.100
  libavformat    57. 83.100 / 57. 83.100
  libavdevice    57. 10.100 / 57. 10.100
  libavfilter     6.107.100 /  6.107.100
  libavresample   3.  7.  0 /  3.  7.  0
  libswscale      4.  8.100 /  4.  8.100
  libswresample   2.  9.100 /  2.  9.100
  libpostproc    54.  7.100 / 54.  7.100
  libpostproc    54.  7.100 / 54.  7.100
[hls,applehttp @ 0x5648cdc12a40] HLS request for url 'https://live.performa.intio.tv/media/0af0186b-112c-4e72-8556-54abedcc7f55/hls_video_6_7800_25init.mp4', offset 0, playlist 0
[hls,applehttp @ 0x5648cdc12a40] Opening 'https://live.performa.intio.tv/media/0af0186b-112c-4e72-8556-54abedcc7f55/hls_video_6_7800_25init.mp4' for reading
Could not read complete segment.
[hls,applehttp @ 0x5648cdc12a40] HLS request for url 'https://live.performa.intio.tv/media/0af0186b-112c-4e72-8556-54abedcc7f55/hls_video_6_7800_25_00001.mp4', offset 0, playlist 0
[hls,applehttp @ 0x5648cdc12a40] Opening 'https://live.performa.intio.tv/media/0af0186b-112c-4e72-8556-54abedcc7f55/hls_video_6_7800_25_00001.mp4' for reading
[h264 @ 0x5648ce135bc0] Reinit context to 1920x1088, pix_fmt: yuv420p
Input #0, hls,applehttp, from 'https://live.performa.intio.tv/media/0af0186b-112c-4e72-8556-54abedcc7f55/hls_video_6_7800_25.m3u8':
  Duration: 02:50:50.72, start: 0.080000, bitrate: 0 kb/s
  Program 0 
    Metadata:
      variant_bitrate : 0
    Stream #0:0: Video: h264 (Main), 1 reference frame (avc1 / 0x31637661), yuv420p(tv, bt709, left), 1920x1080 (1920x1088) [SAR 1:1 DAR 16:9], 7888 kb/s, 25 fps, 25 tbr, 90k tbn, 50 tbc
    Metadata:
      variant_bitrate : 0
Output #0, mp4, to 'file:master-allsubs-master-allsubs.f8685.mp4.part':
  Metadata:
    encoder         : Lavf57.83.100
    Stream #0:0: Video: h264 (Main), 1 reference frame (avc1 / 0x31637661), yuv420p(tv, bt709, left), 1920x1080 (0x0) [SAR 1:1 DAR 16:9], q=2-31, 7888 kb/s, 25 fps, 25 tbr, 90k tbn, 90k tbc
    Metadata:
      variant_bitrate : 0
Stream mapping:
  Stream #0:0 -> #0:0 (copy)
Press [q] to stop, [?] for help
frame=   59 fps=0.0 q=-1.0 size=    2304kB time=00:00:02.28 bitrate=8278.4kbits/frame=   76 fps= 73 q=-1.0 size=    2816kB time=00:00:02.96 bitrate=7793.6kbits/frame=   92 fps= 56 q=-1.0 size=    3584kB time=00:00:03.60 bitrate=8155.7kbits/frame=  108 fps= 50 q=-1.0 size=    4352kB time=00:00:04.24 bitrate=8408.5kbits/frame=  132 fps= 50 q=-1.0 size=    5120kB time=00:00:05.20 bitrate=8066.0kbits/[hls,applehttp @ 0x5648cdc12a40] HLS request for url 'https://live.performa.intio.tv/media/0af0186b-112c-4e72-8556-54abedcc7f55/hls_video_6_7800_25_00002.mp4', offset 0, playlist 0

## Description

WRITE DESCRIPTION HERE
I attempted to download this URL with the comand:
youtube-dl --all-subs https://live.performa.intio.tv/media/0af0186b-112c-4e72-8556-54abedcc7f55/master-allsubs.m3u8

The resulting file has only video and no audio. If used in VLC instead it plays audio but no video.
The original website is: https://www.staatsoperlive.com/0/5ed1c6d7-f688-4c4d-8a24-92109209e463/player
and the followinf m3u8 are present
https://live.performa.intio.tv/media/0af0186b-112c-4e72-8556-54abedcc7f55/master-allsubs.m3u8
https://live.performa.intio.tv/media/0af0186b-112c-4e72-8556-54abedcc7f55/hls_video_6_7800_25.m3u8
https://live.performa.intio.tv/media/0af0186b-112c-4e72-8556-54abedcc7f55/subs-deu.m3u8
https://live.performa.intio.tv/media/0af0186b-112c-4e72-8556-54abedcc7f55/hls_audio_13_96_25.m3u8

I had the exact same experience with another stream from the same website
https://www.staatsoperlive.com/0/9bf554aa-d1d5-45c7-9228-2c16c8822b84/player

And the solution I have at the moment is download the video with youtube-dl and the audio with VLC and combine them afterwards manually.

**Comment by remitamine** [comment]
you have to wait until both the video and audio downloaded and merged.

**Comment by Ethemind** [comment]
That is the problem. At the end of the process there is no audio. I did allow to finish and on the final line it said audio 0

**Comment by remitamine** [comment]
provide the complete verbose log of the process.

**Comment by Ethemind** [comment]
I don't know why but this time downloaded both parts. I did attempt to download the video at least other 3 times before and it is the fist time it download also the audio. I added a note delimited by *** where it used to stop.
I suppose the problem is solved even if I don't know why since I didn't do anything different.

Thanks for the attention and time so far dedicated. I'm still new in reporting issues here and I apologize if I did mistakes along the way.

Attached the log
[Log.txt](https://github.com/ytdl-org/youtube-dl/files/4618856/Log.txt)
